egret/models/acopf.py

```diff
@@ -502,14 +495,13 @@ def create_riv_acopf_model(model_data, include_feasibility_slack=False):
 
     ### fix the reference bus
     ref_bus = md.data['system']['reference_bus']
-    model.vj[ref_bus].fix(0.0)
-    model.vr[ref_bus].setlb(0.0)
-
     ref_angle = md.data['system']['reference_bus_angle']
     if ref_angle != 0.0:
-        raise ValueError('The RIV ACOPF formulation currently only supports'
-                         ' a reference bus angle of 0 degrees, but an angle'
-                         ' of {} degrees was found.'.format(ref_angle))
+        print("IN HERE")
```

Comment:
One final nit..

Reply:
I've created a painful PR for myself. Last nit is gone :)